lona/pytest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from lona import LonaApp
+
+
+class LonaContext:
+    def __init__(self, client, app):
+        self.client = client
+        self.app = app
+
+    def get_url(self):
+        return f'http://{self.client.host}:{self.client.port}'
+
+
+@pytest.fixture()
+async def lona_app_context(aiohttp_client):
```

Comment:
I'm not an expert in async python. Am I right that here it is not necessary?
```suggestion
def lona_app_context(aiohttp_client):
```

Reply:
You are right. I made it a sync function